actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -372,7 +372,7 @@ def []=(name, options)
 
         handle_options(options)
 
-        if @cookies[name.to_s] != value or options[:expires]
+        if @cookies[name.to_s] != value || options[:expires]
```

Comment:
This change is coming from rubocop. I ran rubocop to fix the single quote usage from my change below, causing this.